docker-qgis/qfc_worker/utils.py

```diff
@@ -229,7 +229,7 @@ def open_qgis_project(
     logging.info(f'Loading QGIS project "{project_filename}"…')
 
     if not Path(project_filename).exists():
-        raise FileNotFoundError(project_filename)
+        raise FileNotFoundError(f"File not found: {project_filename}")
```

Comment:
Consider of using  .format? So to be compatible with Internationalization, if needed.

Reply:
This code is running within the worker. There is very low probability we will ever translate the messages there. There is no translation framowork available at all.